src/stabilization_guide.md

```diff
@@ -184,6 +198,10 @@ if something { /* XXX */ }
 ```
 
 [rust-lang/rust#32409]: https://github.com/rust-lang/rust/issues/32409
+[the stabilization chapter of the std dev guide]: https://std-dev-guide.rust-lang.org/feature-lifecycle/stabilization.html
+[the `src/version` file in `master`]: https://github.com/rust-lang/rust/blob/master/src/version
+[the version schedule on the Forge]: https://forge.rust-lang.org/#current-release-versions
+[the Friday before]: https://forge.rust-lang.org/release/process.html
```

Comment:
These link reference names (e.g., `the stabilization chapter of the std dev guide`) are long and prone to breaking if wording is changed. Could you change them to be pithier?

E.g.: `std-guide-stabilization`, `src-version`, `release-process`.

(You'll have to add manual `[...]` link references to the text too.)

Reply:
Good idea.  I've added pithy names to these, and added the `(...)` link references.

(The `[...]` was a typo, right?  My syntax highlighter thinks this is right.)

Reply:
No, it should be `[...]`. For example, `[look at this](src-version)` will link to `src-version` as if it is a URL. On the other hand, `[look at this][src-version]` will use the URL value of the definition `[src-version]: https://example.com` as the link.

Reply:
Oh!  I see; thank you.  Fixed.

Reply:
And I checked the rendered <https://github.com/rust-lang/rustc-dev-guide/blob/library-stabilization/src/stabilization_guide.md> this time to confirm, which I should have done last time but didn't think of.